po/de.po

```diff
@@ -4780,7 +4780,7 @@ msgstr "Buchung ungültig gemacht"
 
 #: ../../gnucash-git/po/../src/gnome/assistant-acct-period.c:191
 msgid "The book was closed successfully."
-msgstr "Der Buchabschluß wurde erfolgreich beendet."
+msgstr "Der Buchabschluss wurde erfolgreich beendet."
```

Comment:
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

Hello,

please change it as described in the diff.

That is new German spelling since 199x.

Am 24.09.2015 um 01:46 schrieb Yawar Amin:
>> @@ -4780,7 +4780,7 @@ msgstr "Buchung ungültig gemacht"
>> 
>> #: ../../gnucash-git/po/../src/gnome/assistant-acct-period.c:191 
>> msgid "The book was closed successfully." -msgstr "Der
>> Buchabschluß wurde erfolgreich beendet." +msgstr "Der
>> Buchabschluss wurde erfolgreich beendet."
> 
> Could the Esszet be left as is?
> 
> --- Reply to this email directly or view it on GitHub: 
> https://github.com/Gnucash/gnucash/pull/35/files#r40272321
> 

- -- 
Mechtilde Stehmann
## Apache OpenOffice.org
## Freie Office Suite für Linux, MacOSX, Windows
## Loook, calender-exchange-provider, libreoffice-canzeley-client
## PGP encryption welcome
## Key-ID 0x141AAD7F
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2

iQIcBAEBCgAGBQJWA4V9AAoJEPKHe7oUGq1/q6QP/2KtvAXqPssobVLJgy9gOKyO
9plwWf617cA+oHrSrY6oiEvbemSXjPvs4BzrVk2dSbh5EB8HrHT0oh1jtw4OfWK1
GuIWGLlW9ItDa5kqrmxhBFMtRrdtumuqJcVoxPI39+hjGwoZFA3zzC7SfCrh7huZ
tv/AjoeXco0zN7477IiV3r33r5NZ9mcx5eMcKtiBu1u4cWD1S+MUFNgR1O4S5LNm
ucrLkPKrfrM2xKzXSDK9D65yONVeuvEhB9GmWmWaEX4XsXp/AkqtA3YP5UxTUmXE
HWqHemNJ3HpWItKnC78BORV8vCaDzb22pDcdwNIj1BfGUmQZZvGYZC89wPG/aYGb
6Ra0Y5GQAm5vAtd4ypb0bhY+mi6LqB0Fq5q54M5DQV+ZGtBOm1lJLicvk2El4W40
kiOsWMVazrWmece1+w/7+7AfiI8uYP0O3mJ0CGpV/Jd36X2BtiGgjUKv6HIaAiai
htx3rm1MGANW2I74LTYpR7LW0ZTbutIyLSo3iZK8BQb5YqgUu7ffaaV/fI9lOQL2
gMJxZegkWPLrLoUb3Lk8bfqdiW7iXpVDiH5I+kcW5RwGpqux/ScfhuGnOLr27hBr
b0eWdM4Js4d4tPGjOKkM6GfL3JZgyYwMh7RiBqPlZ/OevtNv2QJ+LR59nHRehf6q
x9l3nKmJeEpjn6K/mYyI
=MW3L
-----END PGP SIGNATURE-----